requirements-base.txt

```diff
@@ -3,7 +3,8 @@ boto3==1.10.49
 botocore==1.13.50
 celery==4.4.7
 click==7.1.2
-confluent-kafka==1.5.0
+confluent-kafka==1.5.0; python_version < '3.9'
+confluent-kafka==1.6.0; python_version == '3.9'
```

Comment:
This is to help with Linux on Python 3.9:
https://github.com/getsentry/sentry/pull/24394/checks?check_run_id=2097206058
```
/tmp/pip-install-d77_dxr4/confluent-kafka_62fe4d03737747d6bff6dfc1cd8ac8b4/confluent_kafka/src/confluent_kafka.h:22:10: fatal error: librdkafka/rdkafka.h: No such file or directory
       22 | #include <librdkafka/rdkafka.h>
          |          ^~~~~~~~~~~~~~~~~~~~~~
    compilation terminated.
    error: command '/usr/bin/gcc' failed with exit code 1
```